src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -412,6 +412,167 @@ private enum NamespaceParts
             ref NamespaceBodyBuilder body,
             ref SyntaxListBuilder? initialBadNodes,
             SyntaxKind parentKind)
+        {
+            ParseNamespaceBodyWorker(
```

Comment:
this si a call into the normal 'parse namespace body' that has always existed.  The only difference in it now is htat it keeps track if it saw a type-only member (like a method/operator/etc.) when parsing the namespace members.